SwrveConversationSDK/Conversation/SwrveConversationItemViewController.m

```diff
@@ -63,6 +64,86 @@ - (void)viewDidDisappear:(BOOL)animated {
     }
 }
 
+- (void)dealloc {
+    
+    [[NSNotificationCenter defaultCenter] removeObserver:self
+                                                    name:kSwrveNotificationViewReady
+                                                  object:nil];
+    [[NSNotificationCenter defaultCenter] removeObserver:self
+                                                    name:UIDeviceOrientationDidChangeNotification
+                                                  object:nil];
+    [[UIDevice currentDevice] endGeneratingDeviceOrientationNotifications];
```

Comment:
This should definitely be done when the view will disappear, as other view controllers might need to listen to those changes and dealloc might be called when they are active